package.nls.json

```
@@ -101,5 +101,7 @@
     "azureFunctions.validateFuncCoreTools": "Validate the Azure Functions Core Tools is installed before debugging.",
     "azureFunctions.viewCommitInGitHub": "View Commit in GitHub",
     "azureFunctions.viewDeploymentLogs": "View Deployment Logs",
-    "azureFunctions.viewProperties": "View Properties"
+    "azureFunctions.viewProperties": "View Properties",
+    "azureFunctions.functionSubpath": "The default subpath to create new functions.",
+    "azureFunctions.showNodeProgrammingModel": "Enable Node.js (New Model Preview)"
```

"Enable Node.js" makes it sound like we don't support Node.js at all without the setting haha

```suggestion
    "azureFunctions.showNodeProgrammingModel": "Enable preview Node.js programming model"
```